packages/grid/x-data-grid-pro/src/models/gridProSlotProps.ts

```diff
@@ -0,0 +1,6 @@
+import { GridSlotsComponentsProps } from '@mui/x-data-grid/internals';
+import type { GridHeaderFilterCellOverridableProps } from '../components/headerFiltering/GridHeaderFilterCell';
+
+export interface GridProSlotProps extends GridSlotsComponentsProps {
+  headerFilter?: Partial<GridHeaderFilterCellOverridableProps>;
```

Comment:
Yes, see in this demo for example: https://deploy-preview-7760--material-ui-x.netlify.app/x/react-data-grid/filtering/#simple-header-filters

Reply:
Right, I didn't notice that 👍
Should it be renamed to `headerFilterCell` to match the slot name?

Also, it seems to me that the most straightforward way of removing the header filter menu would be:
```tsx
slots={{
  headerFilterMenu: null,
}}
```
Instead of 
```tsx
slotProps={{
  headerFilter: {
    InputComponentProps: {
      headerFilterMenu: null,
    },
  },
}}
```

Reply:
Agreed, though it could still be useful for overriding other props like `InputComponentProps` etc